docs/README.md

```diff
@@ -437,7 +437,78 @@ the program's secret values to local storage, which means they can be read by
 program can start up immediately using cached data, even if the secrets server
 is not reachable when it launches.
 
-## Testing
+## Self-Contained Operation
+
+In some cases, you may need to run a program entirely without access to a
+secrets server. For example, in standalone testing and bootstrapping it may be
+impractical to set up a secrets service, or you may want to deploy the same
+program across different environments where a secrets service may or may not be
+present.
+
+To support these cases, the Go [`setec`][setecpkg] package provides a
+[`FileClient`][fileclient] type that can be plugged into a
+[`setec.Store`][setecstore].  Unlike the normal [`setec.Client`][setecclient],
+a `FileClient` does not use the network at all, but vends secrets read from a
+plaintext JSON file on the local filesystem.
+
+This differs from the cache described in the previous section: With a cache,
```

Comment:
The cache (if specified in config) still gets populated when using FileClient, correct? Asking because I'm picturing that as a pivotal bootstrapping feature, where you can start from a FileClient to seed the cache, and then gracefully pivot to using a server while still having the warm cache during that first restart or three where you're teasing kinks out of the link to the secrets service.

Reply:
Yes, they both work just fine. Let me add a bit about that specifically.

Reply:
I pulled out this paragraph and expanded it a bit, PTAL?